docs/DeveloperGuide.md

```diff
@@ -69,7 +69,7 @@ Eslint and its accompaning modules can be installed through NPM, so do ensure th
 ### Building and running RepoSense from code
 
 1. Execute the following command on the OS terminal inside the project directory. <br/>
-Usage: `gradlew run -Dargs="([-config CONFIG_FOLDER] | -repos REPO_PATH_OR_URL... | -view REPORT_FOLDER) [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]"` <br/>
+Usage: `gradlew run -Dargs="([-config CONFIG_FOLDER] | [-repos REPO_PATH_OR_URL...]) [-view REPORT_FOLDER] [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]"` <br/>
```

Comment:
I think additional bracket is needed in `-view` to denote optional.
E.g. `[-view [REPORT_FOLDER]]`

Reply:
Should also add a sample usage in the dev guide, or at least append the `-view` to one of the sample usage.
Do also update the description such that the report will be open automatically.

Reply:
I have updated the developer guide, please have a look.